### ISSUE TITLE
Latest release contains vulnerable packages

### ISSUE THREAD

**Issue by Eneuman** [actions/runner#3190]
We are using the self hosted runner in our images and our security checks are blinking red because theese vulnerable packages exists in the latest release of the runner:

Runner and hooks are installed like this:
```
# Runner
RUN curl -f -L -o runner.tar.gz https://github.com/actions/runner/releases/download/v${RUNNER_VERSION}/actions-runner-linux-x64-${RUNNER_VERSION}.tar.gz \
    && tar xzf ./runner.tar.gz \
    && rm runner.tar.gz

# Runner hooks
RUN curl -f -L -o runner-container-hooks.zip https://github.com/actions/runner-container-hooks/releases/download/v${RUNNER_CONTAINER_HOOKS_VERSION}/actions-runner-hooks-k8s-${RUNNER_CONTAINER_HOOKS_VERSION}.zip \
    && unzip ./runner-container-hooks.zip -d ./k8s \
    && rm runner-container-hooks.zip
```

A security scan of the image find the following issues.

**Warnings**

[WARNING] [https://github.com/advisories/GHSA-8g9c-28fc-mcx2] [[HELP LINK](https://avd.aquasec.com/nvd/cve-2024-21319)] Package: System.IdentityModel.Tokens.Jwt Installed Version: 5.2.1 Vulnerability CVE-2024-21319 Severity: MEDIUM Fixed Version: 5.7.0, 6.34.0, 7.1.2 Link: \[CVE-2024-21319\]\(https:\/\/avd.aquasec.com\/nvd\/cve-2024-21319\)

[WARNING] [https://github.com/advisories/GHSA-x5qj-9vmx-7g6g] [[HELP LINK](https://avd.aquasec.com/nvd/cve-2019-0657)] Package: System.Private.Uri Installed Version: 4.3.0 Vulnerability CVE-2019-0657 Severity: MEDIUM Fixed Version: 4.3.2 Link: \[CVE-2019-0657\]\(https:\/\/avd.aquasec.com\/nvd\/cve-2019-0657\)

I have checked the source code and the code takes dependency on vulnerable packages.

Any chance theese can be updated in the next release?

**Comment by Boxser1987** [comment]
```
> We are using the self hosted runner in our images and our security checks are blinking red because theese vulnerable packages exists in the latest release of the runner:
> 
> Runner and hooks are installed like this:
> ```
> # Runner
> RUN curl -f -L -o runner.tar.gz https://github.com/actions/runner/releases/download/v${RUNNER_VERSION}/actions-runner-linux-x64-${RUNNER_VERSION}.tar.gz \
>     && tar xzf ./runner.tar.gz \
>     && rm runner.tar.gz
> 
> # Runner hooks
> RUN curl -f -L -o runner-container-hooks.zip https://github.com/actions/runner-container-hooks/releases/download/v${RUNNER_CONTAINER_HOOKS_VERSION}/actions-runner-hooks-k8s-${RUNNER_CONTAINER_HOOKS_VERSION}.zip \
>     && unzip ./runner-container-hooks.zip -d ./k8s \
>     && rm runner-container-hooks.zip
> ```
> 
> A security scan of the image find the following issues.
> 
> **Warnings**
> 
> [WARNING] [https://github.com/advisories/GHSA-8g9c-28fc-mcx2] [[HELP LINK](https://avd.aquasec.com/nvd/cve-2024-21319)] Package: System.IdentityModel.Tokens.Jwt Installed Version: 5.2.1 Vulnerability CVE-2024-21319 Severity: MEDIUM Fixed Version: 5.7.0, 6.34.0, 7.1.2 Link: \[CVE-2024-21319\]\(https:\/\/avd.aquasec.com\/nvd\/cve-2024-21319\)
> 
> [WARNING] [https://github.com/advisories/GHSA-x5qj-9vmx-7g6g] [[HELP LINK](https://avd.aquasec.com/nvd/cve-2019-0657)] Package: System.Private.Uri Installed Version: 4.3.0 Vulnerability CVE-2019-0657 Severity: MEDIUM Fixed Version: 4.3.2 Link: \[CVE-2019-0657\]\(https:\/\/avd.aquasec.com\/nvd\/cve-2019-0657\)
> 
> I have checked the source code and the code takes dependency on vulnerable packages.
> 
> Any chance theese can be updated in the next release?


# - [ ] __[]()[]()__- [ ] ````~~~~`__

**Comment by nebuk89** [comment]
We continue to update our packages, some like .Net have a larger blast radius as some operating systems don't support newer versions so this takes a bit longer. We will keep addressing these and putting out timelines for our major updates.
